broker-daemon/state-machines/fill-state-machine.spec.js

```diff
@@ -626,6 +650,33 @@ describe('FillStateMachine', () => {
     })
   })
 
+  describe('#shouldRetry', () => {
+    let fakeFill
+    let fsm
+
+    beforeEach(() => {
+      fsm = new FillStateMachine({ store, logger, relayer, engines })
+      fakeFill = { error: { message: 'ORDER_NOT_PLACED' } }
```

Comment:
[nit] to make it easier on us if this string changes, we can use the constant value instead of hardcode. If you decide to make this change, you'll have changes in this file and `order-state-machine.spec.js` too

```
const { ORDER_NOT_PLACED } = FillStateMachine.__get__('FILL_ERROR_CODES')
fakeFill = { error: { message: 'ORDER_NOT_PLACED' } }
```